src/ray/util/logging.cc

```diff
@@ -372,9 +392,7 @@ void RayLog::StartRayLog(const std::string &app_name,
       spdlog::drop(RayLog::GetLoggerName());
     }
     auto file_sink = std::make_shared<spdlog::sinks::rotating_file_sink_mt>(
-        JoinPaths(log_dir_, app_name_without_path + "_" + std::to_string(pid) + ".log"),
-        log_rotation_max_size_,
-        log_rotation_file_num_);
+        log_output_fname, log_rotation_max_size_, log_rotation_file_num_);
```

Comment:
With this, we will have rotate by default now?


Thought: `RayLog::StartRayLog` should accept the default rotation max size as a parameter and for gcs it's 0?

Reply:
> With this, we will have rotate by default now?

Yes.

Reply:
> Thought: RayLog::StartRayLog should accept the default rotation max size as a parameter and for gcs it's 0?

That makes sense, fixed.